Packages/vcs/Lib/taylor.py

```diff
@@ -922,15 +922,15 @@ def drawSkill(self, canvas, values, function=None):
         if function is None:
             return
         color = self.skillColor
-        a = MV2.ones((self.detail, self.detail), typecode=MV2.float)
+        a = MV2.ones((self.detail + 1, self.detail + 1), typecode=MV2.float)
```

Comment:
I updated the commit comment. Basically, data was generated with a loop over an right open interval and because of that it was missing the right and top part.